apps/studio/src/components/ErrorBoundary/DefaultServerError.tsx

```diff
@@ -0,0 +1,35 @@
+import NextLink from "next/link"
+import { Flex, Link, Text } from "@chakra-ui/react"
+import { Button } from "@opengovsg/design-system-react"
+
+import { LiftUnderRepair } from "../Svg"
+
+export const DefaultServerError = () => {
+  return (
+    <Flex
+      flexDirection="column"
+      gap="1.5rem"
+      alignItems="center"
+      marginTop="6rem"
+    >
+      <LiftUnderRepair />
+      <Flex flexDirection="column" gap="0.5rem" alignItems="center">
+        <Text textStyle="h5" textAlign="center">
+          Something went wrong
+        </Text>
+        <Text textStyle="body-2" textAlign="center">
+          It's not you, it's us. Please try refreshingthis page.
+          <br />
+          If this issue persists,{" "}
+          <Link variant="inline" href="mailto:support@isomer.gov.sg">
+            contact Isomer admin
```

Comment:
nit: support

```suggestion
            contact Isomer Support
```

Reply:
@seaerchin this one is mb - wasn't aware we were using Support throughout. i'll update Figma too